news/1036.api_change

```diff
@@ -0,0 +1 @@
+Support `additional_parameters` as an optional param in the Submitit launcher plugin.
```

Comment:
I think `plugin` suffix is more appropriate here.

Reply:
Fixed in https://github.com/facebookresearch/hydra/pull/1054/commits/01b1e4337708bd8d8fd8269cb96fa2aaa55057a5